specification/batch/Azure.Batch/tspconfig.yaml

```diff
@@ -16,7 +16,7 @@ options:
     package-dir: "Azure.Batch"
     clear-output-folder: true
     model-namespace: false
-    head-as-boolean: true
+    head-as-boolean: false
```

Comment:
What does this do for HEAD operations that are existence checks such as `poolExists`?

Reply:
The issue I found is that If I make head-as-boolean true then poolExist will return bool so will getFileProperties.  I found it better for both to return a Response then customize them.